MaplyAdapter/OSBNGUtils.m

```diff
@@ -0,0 +1,63 @@
+//
+//  OSBNGUtils.m
+//  Maply
+//
+//  Created by David on 12/01/2016.
+//  Copyright © 2016 Ordnance Survey. All rights reserved.
+//
+
+#import "OSBNGUtils.h"
+@import OSGridPointConversion;
+#import "MaplyComponent.h"
+
+@implementation OSBNGUtils
+
++ (MaplyProj4CoordSystem *)bngCoordinateSystem {
+    NSString *proj4Str = [OSBNGTransformation proj4String];
```

Comment:
As with today's adventures, probably want the non-OSTN02 version here for the time being?